lib/raxx/middleware/pipeline.ex

```diff
@@ -0,0 +1,3 @@
+defmodule Raxx.Middleware.Pipeline do
```

Comment:
Do we actually need this module. It exists only to define a type?

Reply:
No, it should probably be `Middleware.pipeline()`